packages/core/src/generateAppDeclarationFile.test.ts

```diff
@@ -21,7 +21,7 @@ describe("generateAppDeclarationFile", () => {
     test("should parse middleware", () => {
       const parsedApp = generate(
         createTestProgram(`
-        type NextFunction<T = void> = T extends void
+        type SamenNextFunction<T = void> = T extends void
```

Comment:
Ah, nice 👍

Reply:
Yes.... but now I realised we need to rework docs a bit :D